pkg/token/service.go

```diff
@@ -423,72 +438,105 @@ func (d *tokenService) updateToken(key CacheKey, tt mode) error {
 	}
 }
 
-func (ts *tokenService) writeFilesWithRetry(ctx context.Context, maxElapsedTime time.Duration) error {
-	// backoff config with first retry delay of 5s, and then 10s, 20s, ...
-	b := backoff.NewExponentialBackOff()
-	b.InitialInterval = 5 * time.Second
-	b.Multiplier = 2
-	b.MaxElapsedTime = maxElapsedTime
-
-	return backoff.RetryNotify(ts.writeFiles, backoff.WithContext(b, ctx), func(err error, backoffDelay time.Duration) {
-		log.Errorf("Failed to write token files: %s. Retrying in %s", err.Error(), backoffDelay)
-	})
-}
-
-func (d *tokenService) writeFiles() error {
+func (d *tokenService) writeFiles(ctx context.Context, maxElapsedTime time.Duration) []error {
```

Comment:
Can you write a comment for this function what it does?
- Runs go routine for each target output for token, so that each token can be synced in different time manner

Reply:
fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/131/commits/6822928c8eef4cfb5ba60157b5141e6bcf66cfe1

Reply:
@y-myajima Usually in Go lang, you write comments with function name + verb 

Here is a sample:
```go
// RemoveAll removes path and any children it contains.
// It removes everything it can but returns the first error
// it encounters. If the path does not exist, RemoveAll
// returns nil (no error).
// If there is an error, it will be of type *PathError.
func RemoveAll(path string) error {
	return removeAll(path)
}
```

So for this function you should write like:
```
writeFiles outputs every access token and role token cached in the memory.
...

Reply:
sample:
```
/opt/homebrew/Cellar/go/1.22.2/libexec/src/os/path.go
```